platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -523,6 +523,19 @@ OfflineRegion OfflineDatabase::createRegion(const OfflineRegionDefinition& defin
     return OfflineRegion(db->lastInsertRowid(), definition, metadata);
 }
 
+OfflineRegionMetadata OfflineDatabase::updateMetadata(const OfflineRegion& region, const OfflineRegionMetadata& metadata) {
+    // clang-format off
+    Statement stmt = getStatement(
+                                  "UPDATE regions SET description = ?1"
+                                  "WHERE id = ?2");
+    // clang-format on
+    stmt->bindBlob(1, metadata);
+    stmt->bind(2, region.getID());
+    stmt->run();
+    
+    return metadata;
```

Comment:
Does this method need to return the metadata? Under what circumstances would it differ from the metadata that is passed in?